packages/teleport/src/Apps/AddApp/Automatically/Automatically.tsx

```diff
@@ -73,9 +73,7 @@ export default function Automatically(props: Props) {
             </Flex>
             {!cmd && (
               <Text mb="3">
-                Provide the name and URL of your application to generate a
-                script that will automatically install and configure the App
-                service on the server that can access your application.
+                Provide the name and URL of your application to generate our auto-installer script.
```

Comment:
@benarent dont we want to at least tell a user what that script is about to do? If anything to be installed on my machine/box I at least want to know _something_ about what is going to happen.

Reply:
@webvictim what would be your expectations from the user perspective? Will you be more comfortable knowing what this script is about to do?

Reply:
I would obviously like to know exactly what the script does, but I think this box is far too small to explain that in any detail. If somebody really wants to know exactly what the script does (and really they should) then they can just download the URL manually (without running it via `bash`) and read it to see what it does.

If we had the script's source in the OSS repo I'd consider linking to it.